test/autorest/generated/paginggroup/paging.go

```diff
@@ -272,26 +272,8 @@ func (client *pagingOperations) getMultiplePagesFragmentWithGroupingNextLinkHand
 }
 
 // GetMultiplePagesLro - A long-running paging operation that includes a nextLink that has 10 pages
-func (client *pagingOperations) GetMultiplePagesLro(options *PagingGetMultiplePagesLroOptions) (ProductResultPager, error) {
-	req, err := client.getMultiplePagesLroCreateRequest(options)
-	if err != nil {
-		return nil, err
-	}
-	return &productResultPager{
-		client:    client,
-		request:   req,
-		responder: client.getMultiplePagesLroHandleResponse,
-		advancer: func(resp *ProductResultResponse) (*azcore.Request, error) {
-			u, err := url.Parse(*resp.ProductResult.NextLink)
-			if err != nil {
-				return nil, fmt.Errorf("invalid NextLink: %w", err)
-			}
-			if u.Scheme == "" {
-				return nil, fmt.Errorf("no scheme detected in NextLink %s", *resp.ProductResult.NextLink)
-			}
-			return azcore.NewRequest(http.MethodGet, *u), nil
-		},
-	}, nil
+func (client *pagingOperations) BeginGetMultiplePagesLro(options *PagingGetMultiplePagesLroOptions) (ProductResultPager, error) {
```

Comment:
This should return a `ProductResultPoller` (something like that).  When pagers and pollers intersect, the return type is a poller with a pager response type.

Reply:
Just saw the TODO about this.

Reply:
Sounds good, I'll implement it in my next PR